src/itr_evaluator.ml

```diff
@@ -964,7 +977,7 @@ and evaluate_expr (context : 'a context) (e : expr) : record_item =
       | _ -> Rec_value e)
     | _ -> Rec_value e)
   | Value (Funcall { func : value; args : record_item list })
-    when (func = Literal (String "List.length")||func = Literal (String "len")) && List.length args = 1 ->
```

Comment:
The `"len"` function was already handled in the case above, so this case is never hit.